### DISCUSSION TITLE
Grandorgue metronome sound

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#376] [General]
Originally posted 2021-01-26 19:03:41.451000 by supergedeon in GrandOrgue discussion forum on SourceForge<br><br>Hi!
I would like to open a discussion on how to get a more friendly metronome sound using GrandOrgue.
Thank you in advance for your feedback.
Alain
<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962124/thread/bf1dbefad5/28c3/attachment/alternate<br>

**Comment by supergedeon** [comment]
Hi!
Back on the topic of getting cooler sounds for the metronome.
Under Windows, I found somewhere that metronome sounds where located in GrandOrgue/Share/GrandOrgue/sounds/metronome.
They appeared to be "beat.wv" and "firstbeat.wv". 
Originally, they are awfull bips. I found it possible to replace them by more pleasant clics, just by changing the extension of .wav files.

I recently updated to version 3.12.0-1 and those files are not taken into account anymore. If I change the names "beat" and "firstbeat", I still get the nasty "bips". A searched for .wv files in all the computer returned nothing appart the original files.

Question: are metronome sounds now hard coded in the generator's routine, or did their name changed, or are they located elsewhere?

Thank you in advance for your return.

**Comment by larspalo** [comment]
It might depend on what file you've actually downloaded, but I can still find the metronome files in the .zip package for windows in the latest release here on GitHub. If you downloaded the .exe file instead I have no idea where they might have ended up...

**Comment by supergedeon** [comment]
Thank you Lars for your answer. I actually loaded the .exe package.
I tried to rename "metronome" directory and the sounds remaned the same .... I don't remember GrandOrgue's previous version behavior, but it was clear that the beat files are not used in real time yet. So I rebuilt the cache of a peculiar organ (GrandOrgue's demo) after having substituted the click files with mine, and that was it! The metronome files are loaded with each organ separately as the other sounds are (pipes, noises ..). One can see the related instructions at the end of the list in the description of the presently loaded organ. Man muss wissen ....

And my congratulations to all the active developpers who contribute to this wondefull project I've been using for 15 years now!
